packages/ag-charts-community/src/scene/shape/shape.ts

```diff
@@ -67,13 +81,14 @@ export abstract class Shape<D = any> extends Node<D> {
     strokeOpacity: number = 1;
 
     @SceneChangeDetection({ changeCb: (s: Shape) => s.onFillChange() })
-    fill?: string | Gradient = Shape.defaultStyles.fill;
+    fill?: string | AgGradientFill | Gradient = Shape.defaultStyles.fill;
```

Comment:
I'm seeing the type `string | AgGradientFill | Gradient` repeated a lot, could we add a type-alias for this union to centrally define this type?